apps/abstraxion-dashboard/hooks/useAbstraxionAccount.ts

```diff
@@ -35,33 +46,25 @@ export const useAbstraxionAccount = () => {
     const refreshConnectionType = () => {
       if (session) {
         setConnectionType("stytch");
-      } else if (data) {
+      } else if (isConnected) {
+        // Is this the right conditional
```

Comment:
is this still an open question?

Reply:
This gets cleaned up in the eth authenticator PR, but this does work for this PR